pkg/ccl/streamingccl/streamproducer/event_stream.go

```diff
@@ -191,6 +191,20 @@ func (s *eventStream) onInitialScanDone(ctx context.Context) {
 	s.addMu = nil
 }
 
+func (s *eventStream) onValues(ctx context.Context, values []kv.KeyValue) {
```

Comment:
we need to pass this into rangfeed init here, yeah?

Reply:
huh, yeah, apparently I dropped a commit in a rebase, because I had a nice `WithOnValue` and everything, with a big comment about overhead and options, and can't find it anywhere now. re-added.